itou/metabase/management/commands/sql/021_tx_candidats_acceptes_par_prescripteurs.sql

```diff
@@ -0,0 +1,17 @@
+select 
+    origine_détaillée, 
+    date_candidature, 
+    count(distinct(id_candidat_anonymisé)) filter (where(état = 'Candidature acceptée')) as nombre_candidats_acceptés,
+    count(distinct(id_candidat_anonymisé)) as nombre_candidats,
+    count(distinct(id_anonymisé)) filter (where(état = 'Candidature acceptée')) as nombre_candidatures_acceptées,
+    count(distinct(id_anonymisé)) as nombre_candidatures
+from 
+    candidatures
+where 
+    (origine <> 'Candidat' or origine is null)
+    and (origine <> 'Employeur' or origine is null) 
```

Comment:
`(A or B) and (C or B)` équivalent à `B or (A and C)` non ?

Reply:
En fait quelles sont les valeurs possibles ? Au final tu veux pas juste un `origine = 'Prescripteur'` ou un truc comme ça ?

Reply:
Ton code actuel prend en compte les `origine is null` c'est normal ? Tu ne voudrais pas plutôt prendre en compte uniquement les candidatures prescripteur ?

Reply:
En fait c'est une requête initialement faite sur Metabase que j'ai transformé sur l'outil en SQL.
Ça a complexifié la requête.
L'objectif est de récupérer les origines prescripteurs